### PR TITLE
Add de.willuhn.Jameica

### PR DESCRIPTION
**Pull request by tobias-hammerschmidt**
Initial setup for Jameica application platform

### CODE REVIEW

**Comment by tobias-hammerschmidt** [comment]
bot, build de.willuhn.Jameica

**Comment by tobias-hammerschmidt** [comment]
bot, build de.willuhn.Jameica

**Comment by nedrichards** [comment]
@tobias-hammerschmidt have you contacted upstream about adding the app to flathub?

File: de.willuhn.Jameica.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>de.willuhn.Jameica</id>
+  <name>Jameica</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <developer_name>Olaf Willuhn</developer_name>
+  <summary>Application-Platform similar to OSGI</summary>
+  <description><p>Jameica is an Application-Platform written in Java containing a SWT-UI. It provides different services (GUI-Toolkit, Logging, Security, Backup, Lifecycle-Management, Message-Bus) to the installed plugins. It's a kind of runtime environment similar to OSGi.</p></description>
```
Comment (nedrichards) [inline de.willuhn.Jameica.appdata.xml]: Given that this app is only available in german it'd be good to have localized summary and descriptions in that language.

**Comment by nedrichards** [comment]
Also, it'd be good to send the appdata upstream, all Linux systems benefit from sharing good quality metadata there.

**Comment by tobias-hammerschmidt** [comment]
> @tobias-hammerschmidt have you contacted upstream about adding the app to flathub?

Yes I've contacted upstream before opening the pull-request but the developer is not interested in providing a flatpak version. However he didn't mind if I provide it.

File: de.willuhn.Jameica.appdata.xml (hunk 1 above)
Comment (tobias-hammerschmidt) [reply de.willuhn.Jameica.appdata.xml]: Good point - I've updated the app metadata.

**Comment by tobias-hammerschmidt** [comment]
> Also, it'd be good to send the appdata upstream, all Linux systems benefit from sharing good quality metadata there.

Also a valid point. Upstream only provides the app in form of a zipped archive and I don't think adding the metadata to the zip will improve the situation for packagers of various distros. However let me try to reach out to the packagers listed in https://www.willuhn.de/wiki/doku.php?id=support:bezugsquellen maybe they are interested in reusing the app metadata.

**Comment by tobias-hammerschmidt** [comment]
bot, build de.willuhn.Jameica

**Comment by nedrichards** [comment]
@tobias-hammerschmidt you need a launchable tag in the appdata per https://github.com/flathub/flathub/wiki/AppData-Guidelines it should say:
> <launchable type="desktop-id">de.willuhn.Jameica.desktop</launchable>

**Comment by tobias-hammerschmidt** [comment]
bot, build de.willuhn.Jameica

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/de.willuhn.Jameica

**Pull request closed without merging**